enhancements/kubernetes-nmstate-integration.md

```diff
@@ -124,6 +144,8 @@ Initial support for kubernetes-nmstate will be Tech Preview
 
 #### Tech Preview -> GA
 
+- kubernetes-nmstate handler can be installed by kubernetes-nmstate-operator
```

Comment:
How is this related to the graduation? We need the operator to be able to install knmstate in OCP even in TP.

Reply:
True. I'll remove.